src/components/asset-panel/selector.jsx

```diff
@@ -55,7 +56,11 @@ const Selector = props => {
 
 Selector.propTypes = {
     buttons: PropTypes.arrayOf(PropTypes.shape({
-        text: PropTypes.string.isRequired,
+        message: PropTypes.shape({
+            id: PropTypes.string.isRequired,
+            defaultMessage: PropTypes.string,
+            description: PropTypes.string
+        }),
```

Comment:
This API seems unintuitive to me, since it makes localization information a part of the spec of the component. For components in the future, could we keep the `text` prop, but make a new PropType that can be either a string or a `FormattedMessage` component (e.g. `text: CustomPropTypes.localizedString`)?  I think that would help organizationally, since it would keep the localization info contained to the place it's used — here we have to update the component tree to account for it.